.github/workflows/label-conflict.yml

```diff
@@ -4,32 +4,26 @@ on:
     branches:
       - 'develop'
       - '4.*'
+    types:
+      - labeled
 
 concurrency:
   group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
   cancel-in-progress: true
 
-permissions:
-  contents: read
-  issues: write
-  pull-requests: write
-
 jobs:
-  auto-label:
+  auto-label-conflict:
+    if: github.event.label.name == 'stale'
     runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
     steps:
-      - uses: prince-chrismc/label-merge-conflicts-action@v3
+      - name: Add comment for PR with conflict
+        uses: peter-evans/create-or-update-comment@v3
         with:
-          conflict_label_name: 'stale'
-          github_token: ${{ github.token }}
-
-          # --- Optional Inputs ---
-          # To make sure the merge commit exactly matches the branch
-          detect_merge_changes: true # or true to handle as conflicts
-          # By default a comment will be left, adding `conflict_comment: ''` will disable comments
-          # The optional `${author}` will be replaced with the username of the pull request
-          conflict_comment: |
-            :wave: Hi, @${author},
+          issue-number: ${{ github.event.pull_request.number }}
+          body: |
+            :wave: Hi,
```

Comment:
I think you can still get the author's name?
`${{ github.event.pull_request.user.login }}`

Reply:
Done